e2e/app/page/google-login.ts

```diff
@@ -109,7 +109,7 @@ export default class GoogleLoginPage {
       submitButton.click(),
```

Comment:
Can you add some comments to these lines of code explaining why we've chosen the associated timeout? As a reviewer, I have no idea why they were set to their previous values *or* why we are changing them now. Comments would take care of that – explaining the "why" of the code when it's not obvious from the context.